forem/forem
# Remove Collective Noun-Related Fields from SiteConfig

## What type of PR is this? (check all applicable)


## Description
This PR does the following:
- Adds a `data_update` script that destroys `collective_noun` and `collective_noun_disabled`
- Adds a spec for the `data_update` script that destroys `collective_noun` and `collective_noun_disabled`

**Heads Up**: This PR is on pause until its previous, related PRs are deployed to the Forem fleet! I will open this PR up for review once all Forems have been updated.

## Related Tickets & Documents
Relates to PR #11846 and PR #12055 

## QA Instructions, Screenshots, Recordings
To QA this PR, please ensure that all of the checks pass and that Travis builds green. Additionally, you can run this `data_update` script manually to test that it works as expected.

### UI accessibility concerns?
N/A

## Added tests?


## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [x] Removed references to "Collective Noun" and "Collective Noun Disabled" from the Admin guides in a previous PR

## [optional] Are there any post-deployment tasks we need to perform?
The `data_update` script in this PR needs to successfully run! ♻️ 

## [optional] What gif best describes this PR or how it makes you feel?

![Kevin Talking With A Few Words](https://media.giphy.com/media/DMNPDvtGTD9WLK2Xxa/giphy.gif)
